changelog/11911.bugfix.md

```diff
@@ -0,0 +1 @@
+Changed categorical slot comparison to be case insensitive.
```

Comment:
I actually just realised this PR targets `main`, but bug-fixes should be merged to the release branch (e.g. `3.4.x`), is it possible to please rebase?

Reply:
I did a rebase, but not sure if it worked. I suggest we close this PR, and I make a fresh PR with a new branch

Reply:
That works 👍🏻

Reply:
Oh, try first to edit the target branch in the UI here